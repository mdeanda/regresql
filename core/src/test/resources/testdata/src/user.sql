select id, username
--
from user_account limit 1;
