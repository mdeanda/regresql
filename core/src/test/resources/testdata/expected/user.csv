"id","username"
"1","mdeanda"
